test/functional/swupdlib.bash

```diff
@@ -105,7 +105,18 @@ check_lines() {
 
   if [ $ret -eq 1 ]; then
     echo -e "\nChecked lines versus actual output (note that actual output may contain ignored lines):\n"
-    diff -u "$checked" "$outputfile"
+    # Omit ignored lines in the actual output to unobfuscate test error
+    # reporting.
+    diff -u "$checked" \
```

Comment:
2 comments. Normally one would use the NR == FNR idiom here (subject to trhe ignored file not being empty, or else just read the ignored file in a BEGIN clase), e.g.

awk 'NR == FNR { toignore[$0]=1 ; next} { for (i in toignore) { if ($0 ~ i) { next } } print $0 }' "$ignored" "$outputfile"

but I think the tool to use is grep, i.e.

grep -Evf "$ignored" "$output"

Reply:
Ah, yes, thanks for the feedback. I will switch to using grep.